### PR TITLE
Add function to compare CPU count in BIOS Data to CPU count of machine

### PR DESCRIPTION
**Pull request by ChriMarMe**

### CODE REVIEW

**Comment by ChriMarMe** [comment]
#54 can be closed.

File: pkg/test/memory.go [hunk 1]
```diff
@@ -386,6 +394,13 @@ func TestBIOSDATAREGIONValid() (bool, error) {
 	return true, nil
 }
 
+func TestBIOSDATACPUCOUNTValid() (bool, error) {
+	if biosdata.NumLogProcs != api.CPULogCount() {
+		return false, fmt.Errorf("Logical CPU count in BIOS and machine doesnt match")
```
Comment (PatrickRudolph) [inline pkg/test/memory.go]: BIOS => BIOSDATA
machine -> CPUID

File: pkg/test/memory.go [hunk 2]
```diff
@@ -76,6 +76,13 @@ var (
 		dependencies: []*Test{&testbiosdataregionpresent},
 		Status:       TestImplemented,
 	}
+	testbiosdatacpucountvalid = Test{
+		Name:         "Bios/Machine CPU count matches",
```
Comment (PatrickRudolph) [inline pkg/test/memory.go]: BIOS DATA NumLogProcs valid

Comment (ChriMarMe) [reply pkg/test/memory.go]: done

File: pkg/test/memory.go (hunk 1 above)
Comment (ChriMarMe) [reply pkg/test/memory.go]: done

**Pull request closed without merging**